websites/C/Currys/presence.ts

```diff
@@ -0,0 +1,91 @@
+const presence = new Presence({
+		clientId: "996516483150663721",
+	}),
+	{ pathname, host } = document.location,
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData = {
+		smallImageKey: "search",
+		largeImageKey: "logo",
```

Comment:
Imgur links are used instead of Discord asset names
```suggestion
		smallImageKey: "search",
		largeImageKey: "https://i.imgur.com/yOolF4f.png",
```

Reply:
i thought I changed that already guess not, I'll get that changed thanks.